### DISCUSSION TITLE
Weather Impact on tours

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#917] [Ideas]
Today on a Strava activity, I found this "myWindsock report" that shows how the weather impacted the ride of the Strava user.

It made me wonder if MT could generate a similar report considering that I've now added weather retrieval in MT since 2019.

This is the report I found:

_-- myWindsock Report --
Weather Impact: -21.2%
Headwind: 62% @ 2.6-22.5mph
Longest Headwind: 01h 51m 49s
Air Speed: 16.1mph
Temp: 58.3-85.8°F
Precip: 0% @ 0 Inch/hr
-- END --_

What do you think ?
I  ride bikes but mainly run so maybe @wolfgang-ch and @telemaxx , you could identify relevant data?

More details on their [website](https://mywindsock.com/page/help/features/)

**Comment by wolfgang-ch** [comment]
With weather reports from distributed weather stations you will never get precise data (head/tail wind) which is mainly influencing the tour. 

Firstly I would install a power meter before using weather reports.

**Comment by wolfgang-ch** [comment]
When you would have power data then you could create a report (graphical/summary) how it influenced the tour parts. But I've never had a power meter why I didn't created this type of report.

**Reply by FJBDev** [reply]
Me neither, I don't have a power meter

**Reply by stvzldn** [reply]
I use a Velocomp Newton powermeter which measures various forces including windspeed.  Their analysis program has a tool called Effort Speed which graphs the effect of headwind/tailwind on the rider speed.  I have not found this to be very useful.  Of greater use is their measurement of extraneous motion and pedal forces to compute wasted watts using data not available to others.
